src/planner/rules/plan.rs

```diff
@@ -18,7 +18,7 @@ pub fn always_better_rules() -> Vec<Rewrite> {
 
 #[rustfmt::skip]
 fn cancel_rules() -> Vec<Rewrite> { vec![
-    rw!("limit-null";       "(limit null 0 ?child)"     => "?child"),
+    rw!("limit-null";       "(limit null 0 ?child)"     => "(empty ?child)"),
```

Comment:
The definition of the limit node is `(limit limit_num offset_num child)`.
So `(limit null 0 ?child)` means `LIMIT ALL OFFSET 0`, which is a dummy operation.
Actually this is not a bug. 😄

Reply:
Thanks for the reminder, but limit 0 actually returns a logical plan similar to
`LogicalValues(tuples=[[]]).`
 I don't know if risinglight has implemented it now. If not, I'd be happy to contribute.